docs/src/PageHeader.doc.js

```diff
@@ -0,0 +1,645 @@
+                    onDismiss={() => {
+                      setOpen(false);
+                    }}
+                  >
+                    <Dropdown.Section label="Categories">
+                      <Dropdown.Item
+                        handleSelect={handleSelect}
+                        selected={selected}
+                        option={{ value: 'item 1', label: 'Item 1' }}
+                      />
+                      <Dropdown.Item
+                        handleSelect={handleSelect}
+                        selected={selected}
+                        option={{
+                          value: 'item 2',
+                          label: 'Item 2 with a really long, detailed, complex name',
+                        }}
+                      />
+                    </Dropdown.Section>
+                    <Dropdown.Section label="View options">
+                      <Dropdown.Item
+                        handleSelect={handleSelect}
+                        selected={selected}
+                        isExternal
+                        href="https://pinterest.com"
+                        option={{
+                          value: 'item 3',
+                          label: 'Item 3 with a really long, detailed, complex name',
+                        }}
+                      />
+                      <Dropdown.Item
+                        handleSelect={handleSelect}
+                        selected={selected}
+                        badgeText="New"
+                        option={{ value: 'item 4', label: 'Item 4' }}
+                      />
+                      <Dropdown.Item
+                        handleSelect={handleSelect}
+                        selected={selected}
+                        isExternal
+                        href="https://pinterest.com"
+                        badgeText="New"
+                        option={{
+                          value: 'item 5',
+                          label: 'Item 5 with a really long name is a new item!',
+                        }}
+                      />
+                    </Dropdown.Section>
+                  </Dropdown>
+                )}
+              </React.Fragment>
+            }
+            primaryAction={<Button color="red" text="Create product group" />}
+          />
+        );
+      }
+    `}
+  />,
+);
+
+// Specifying Component gives auto warnings about missing props
+card(
+  <PropTable
+    Component={PageHeader}
+    props={[
+      {
+        name: 'title',
+        type: 'string',
+        required: true,
+        description:
+          'Page title, will always be an level 1 heading. Content should be [Localized](/Localized).',
+      },
+      {
+        name: 'maxWidth',
+        type: `number | string`,
+        description: `Use numbers for pixels: maxWidth={100} and strings for percentages: maxWidth="100%."`,
+      },
+      {
+        name: 'primaryAction',
+        type: `React Node`,
```

Comment:
These should be `React.Node`, though I think we should type more specifically for the valid components we accept. It's not a fair API to have secretly valid types that aren't reflected in the docs.